src/helpers/geoLocation.ts

```diff
@@ -0,0 +1,54 @@
+import cloneDeep from 'lodash/cloneDeep';
+import isEmpty from 'lodash/isEmpty';
+import set from 'set-value';
+import { FixMe } from '../util/types';
+import { getFirstMatchingKeyAndValue } from '../v0/util';
+import * as GenericFieldMappingJson from '../v0/util/data/GenericFieldMapping.json';
+
+export default class GeoLocationHelper {
```

Comment:
any reason to use default class?

Reply:
Resolving this comment, as we are ok with usage of default class in this case.